lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -8,6 +8,7 @@
 from pprint import pformat
 from DataFileUtil.DataFileUtilClient import DataFileUtil
 from DataFileUtil.baseclient import ServerError as DFUError
+from kb_ea_utils.kb_ea_utilsClient import kb_ea_utils
```

Comment:
There's still 5-10 or so pep8 errors in the impl and test files, mostly due to no whitespace between # and the comment, although there are a couple of bad indents